test/unity/toolkits/activity_classifier/test_activity_classifier.cxx

```diff
@@ -0,0 +1,284 @@
+/* Copyright © 2018 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+#define BOOST_TEST_MODULE test_activity_classifier
+
+#include <boost/test/unit_test.hpp>
+#include <core/util/test_macros.hpp>
+#include <iostream>
```

Comment:
What is this header used for?

If we need it, then system headers should precede project headers
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes